blosc2/SChunk.py

```diff
@@ -315,6 +315,117 @@ def update_data(self, nchunk, data, copy):
         blosc2_ext._check_access_mode(self.urlpath, self.mode)
         return super(SChunk, self).update_data(nchunk, data, copy)
 
+    def get_slice(self, start=0, stop=None, out=None):
+        """Get a slice from :paramref:`start` to :paramref:`stop`.
+
+        Parameters
+        ----------
+        start: int
+            The nth item where the slice will begin. Default is 0.
```

Comment:
nth item -> index

Reply:
Done in cae9aacbc84bbde8df87dbe260a9a8a33f83dc9f.